schema/revert/computed_columns/full_name.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert cif:computed_columns/full_name from pg
+
+BEGIN;
+
+drop function cif.full_name(anyelement);
```

Comment:
```suggestion
drop function cif.cif_user_full_name(cif.cif_user);
```

This file also needs to be renamed to match the change name